proxy/core/acceptor/local.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+
+    .. spelling::
+
+        acceptor
+"""
+import os
+import queue
+import socket
+import logging
+import argparse
+import selectors
+import threading
+import multiprocessing.synchronize
+
+from multiprocessing import connection
+from typing import Optional, Tuple, List, Dict
+from typing import Any      # noqa: W0611   pylint: disable=unused-import
```

Comment:
Such things were again annoying.  I had to disable this for `flake8` and `pylint` separately :(.  Another redundancy imho.  We should not check for common errors across `flake8` and `pylint`, specially if it disrupts the smooth development flow.

Reply:
It's hard to do this because many of the checks are not 100% equivalent.

Reply:
Also, you could just put it under an if-block as per common usage pattern
```suggestion
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from typing import Any
```
and you won't have these linting violations. The linters are correct: you do not currently refer this name in runtime.

Reply:
Is there a `--fix` equivalent for these which will automatically add the suggestion instead of throwing errors?

Reply:
This didn't work, ran into runtime errors:

```console
File "/Users/abhinavsingh/Dev/proxy.py/proxy/core/acceptor/local.py", line 32, in LocalExecutor
    def __init__(self, *args: Any, **kwargs: Any) -> None:
NameError: name 'Any' is not defined. Did you mean: 'any'?
```

Reply:
> Is there a `--fix` equivalent for these which will automatically add the suggestion instead of throwing errors?

No, linters exist to point out problems. They don't have a human context to make decisions on how you want to address the problems you've created.

Reply:
> This didn't work, ran into runtime errors:
> 
> ```
> File "/Users/abhinavsingh/Dev/proxy.py/proxy/core/acceptor/local.py", line 32, in LocalExecutor
>     def __init__(self, *args: Any, **kwargs: Any) -> None:
> NameError: name 'Any' is not defined. Did you mean: 'any'?
> ```

This means that you've actually started using the variable and don't need a conditional import. Now you need it to be unconditional and there's no violation you had earlier.

Reply:
Yep `Any` is part of most constructor.

Reply:
(not that it's a good idea, though)